presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -228,6 +228,15 @@ Statistical Aggregate Functions
 
     Returns the sample covariance of input values.
 
+.. function:: kurtosis(x) -> double
+
+    Returns the (excess) kurtosis of all input values. Unbiased estimate using
```

Comment:
Could you update this to remove the parentheses around "excess" because this function only generates the excess kurtosis?